qdrant/runtime/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-qdrant-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-langchain4j-qdrant</artifactId>
+    <name>Quarkus LangChain4j - Qdrant embedding store - Runtime</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.langchain4j</groupId>
+            <artifactId>quarkus-langchain4j-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-qdrant</artifactId>
```

Comment:
Ideally we would want to avoid this altogether as it forces us use a different network stack altogether. 

I suppose we could use it for now as a first version, but I would really like us to move to our own client (which has a bunch of benefits)

Reply:
Yeah... I do agree, but...
But the thing is that if we implement our own client, we bring upon ourselves a huge maintenance burden. :/
With the upstream impl, we simply get future fixes "for free". I can't imagine us maintaining separate clients for everything that langchain4j supports.

For Milvus, we currently also use the upstream (gRPC-based) client. I tried creating our own, but it turned out to be quite a nightmare.

Reply:
In this case it should be much easier as there is also a REST API.

But I guess we can leave it for another time...

Reply:
If we use the upstream client we would not have metrics, tracing, security, context propagation and so on. Basically, we would bring nothing.

Reply:
You may even create too many event loops.

Reply:
I don't have any objection about using our own client, but I don't have enough knowledge of the underlying langchain4j implementation to asses how complex and maintainable it would be.